buildSrc/subprojects/kotlin-dsl/src/main/kotlin/build/PatchKotlinCompilerEmbeddable.kt

```diff
@@ -31,17 +32,12 @@ import org.gradle.api.tasks.OutputFile
 import org.gradle.api.tasks.PathSensitive
 import org.gradle.api.tasks.PathSensitivity
 import org.gradle.api.tasks.TaskAction
-
-import org.gradle.api.internal.file.archive.ZipCopyAction.CONSTANT_TIME_FOR_ZIP_ENTRIES
-import org.gradle.api.internal.file.pattern.PatternMatcherFactory
-
+import org.gradle.kotlin.dsl.*
 import java.io.File
 import java.util.zip.ZipEntry
 import java.util.zip.ZipFile
 import java.util.zip.ZipOutputStream
 
-import org.gradle.kotlin.dsl.*
-
 
```

Comment:
Nit: double new-line

Reply:
My understanding is that a double-line between imports and type declarations is part of the coding standard.